src/write.c

```diff
@@ -3043,9 +3039,12 @@ static avifResult avifRWStreamWriteProperties(avifItemPropertyDedup * const dedu
 
             // Based on the explanation above, 'clap', 'irot' and 'imir' have to match between the base and
             // gain map image items in the container part of the encoded file.
-            // To enforce that, the transformative properties of the gain map cannot be set explicitly in the API.
+            // 'pasp' is not a transformative property (despite AVIF_TRANSFORM_PASP being part of
+            // avifTransformFlag) but it is assumed to apply to the gain map in the same way as
+            // the transformative properties above.
```

Comment:
This sentence (lines 3042-3044) should not be added to the middle of this paragraph. It should go before this paragraph. I moved this sentence forward in https://github.com/AOMediaCodec/libavif/pull/2528.

Reply:
Sorry about that, I tried to respect the given line numbers but I was not sure they took other modifications into account. Thank you for the PR.

Reply:
No worries! I thought that might be why. I will try to provide some context for a location in a file in the future.